sky/core.py

```diff
@@ -760,10 +752,9 @@ def job_status(
     if job_ids is not None and len(job_ids) == 0:
         return {}
 
-    if not silent:
-        sky_logging.print(f'{colorama.Fore.YELLOW}'
-                          'Getting job status...'
-                          f'{colorama.Style.RESET_ALL}')
+    sky_logging.print(f'{colorama.Fore.YELLOW}'
```

Comment:
sorry why do we remove if not slient?

Reply:
The `silent` argument is introduced in skyserve implementation to avoid corrupt command line output. Since we no longer use `core` API, we could restore it to the same version with master

Reply:
got it.